[FIX] account_asset_management:

FernandoRomera opened the pull request:
* multicompany error

FernandoRomera committed:
[FIX] account_asset_management:
* multicompany error

astirpe commented on account_asset_management/migrations/13.0.1.0.0/post-migration.py:
```diff
@@ -0,0 +1,8 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from openupgradelib import openupgrade  # pylint: disable=W7936
```
The name of the migration directory should be `13.0.1.1.3` or higher, otherwise updating this module will not trigger this migration script.
Moreover, here you are assuming that the `openupgrade` lib is already installed before the module is updated, which is not always the case. You could avoid `openupgrade`, for example like it was done here: https://github.com/OCA/queue/blob/13.0/queue_job/migrations/13.0.1.2.0/post-migrate-queue-job.py

FernandoRomera replied:
In https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0, say:

Multi-company record rules now need to have this general form: ['|',('company_id','=',False),('company_id','in',company_ids)]</field>, and you need to add migration script as well when being noupdate="1" record (which should be the majority). See https://github.com/OCA/credit-control/pull/54/commits/162cde23fe5c0dd12f7de1fa01d36a0bc156008e for an example.

And the example use  openupgrade lib.

astirpe replied:
Yes, but you are not migrating to V13 here. You are bug-fixing an already migrated module. The most of the users of this module V13 do not have `openupgrade` lib installed.

FernandoRomera committed:
[FIX] version

[FIX] new line

[FIX] new line

[FIX] without openupgrade

[FIX] without openupgrade

[FIX] without openupgrade

[FIX] without openupgrade

[FIX] without openupgrade

astirpe commented on account_asset_management/migrations/13.0.1.1.3/post-migration.py:
```diff
@@ -0,0 +1,27 @@
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
```
License should stay AGPL

astirpe reviewed (approved):
Looks good. When travis will be green, can you squash all the commits into just one?

MRomeera reviewed (approved):
Ok

pedrobaeza commented:
/ocabot merge minor
